engine/src/components/afterburner.h

```diff
@@ -41,15 +41,14 @@ class Afterburner : public Component, public EnergyConsumer {
     Afterburner(EnergyContainer *source = nullptr);
     
     // Component Methods
-    virtual void Load(std::string upgrade_key, 
-                      std::string unit_key = "");      
+    virtual void Load(std::string unit_key);      
```

Comment:
If these are overloads, they should be marked with `override` at the end, rather than with `virtual`.

Reply:
Sorry, I misspoke. Not overloads -- overrides. These methods, along with many others in this PR, override methods inherited from parent classes. As such, they should be marked `override` rather than `virtual`. (`virtual` should be used only in the first base class that declares each method.)

Reply:
Oh, now I see. This change has been implemented already, across quite a few files. Looking good. I don't know why I didn't see that before.